server/tserver/src/main/java/org/apache/accumulo/tserver/session/SessionManager.java

```diff
@@ -65,10 +66,10 @@
   private final Long expiredSessionMarker = (long) -1;
   private final AccumuloConfiguration aconf;
 
-  public SessionManager(AccumuloConfiguration conf) {
-    aconf = conf;
-    maxUpdateIdle = conf.getTimeInMillis(Property.TSERV_UPDATE_SESSION_MAXIDLE);
-    maxIdle = conf.getTimeInMillis(Property.TSERV_SESSION_MAXIDLE);
+  public SessionManager(ServerContext context) {
+    this.aconf = context.getConfiguration();
```

Comment:
This comment is not about this particular change, but more about all of the changes that switched from passing config to passing context.  Did you verify that all of these types of changes are still using the same config type?  For example if it used to be the case that table or site config were passed in, is it getting the same thing from the context?

Reply:
So, I think this specific instance is ok. In most cases, the AccumuloConfiguration being passes is from `context.getConfiguration`. I did revert a couple of cases where I could not be sure, DistributedWorkQueue and AssignmentWatcher.